server/item/enchantment/frost_walker.go

```diff
@@ -0,0 +1,49 @@
+package enchantment
+
+import (
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+)
+
+// FrostWalker is an enchantment to boots that creates frosted ice blocks when walking
+// over water, and causes the wearer to be immune to damage from blocks such as magma
+// blocks and campfires when stepped on.
+type FrostWalker struct{}
+
+// Name ...
+func (FrostWalker) Name() string {
+	return "Frost Walker"
+}
+
+// MaxLevel ...
+func (FrostWalker) MaxLevel() int {
+	return 2
+}
+
+// Cost ...
+func (FrostWalker) Cost(level int) (int, int) {
+	return 25, 50
```

Comment:
```suggestion
	min := level * 10
	return min, min + 15	
```

Reply:
Why did you choose these values?

I'm not sure if it matters, but frost walker is not an enchantment available in enchanting tables.

Reply:
https://minecraft.fandom.com/wiki/Enchanting/Levels

It affects the amount of XP the grindstone gives back

Reply:
Gotcha